src/platforms/browser/WebPlatform.mjs

```diff
@@ -112,6 +113,109 @@ export default class WebPlatform {
         return cancelCb;
     }
 
+    createRoundRect(cb, stage, w, h, radius, strokeWidth, strokeColor, fill, fillColor) {
+        if (fill === undefined) fill = true;
+        if (strokeWidth === undefined) strokeWidth = 0;
+
+        let canvas = this.getDrawingCanvas();
+        let ctx = canvas.getContext('2d');
+        ctx.imageSmoothingEnabled = true;
+
+        canvas.width = w + strokeWidth + 2;
+        canvas.height = h + strokeWidth + 2;
+
+        ctx.beginPath();
+        let x = 0.5 * strokeWidth + 1, y = 0.5 * strokeWidth + 1;
+
+        ctx.moveTo(x + radius[0], y);
+        ctx.lineTo(x + w - radius[1], y);
+        ctx.arcTo(x + w, y, x + w, y + radius[1], radius[1]);
+        ctx.lineTo(x + w, y + h - radius[2]);
+        ctx.arcTo(x + w, y + h, x + w - radius[2], y + h, radius[2]);
+        ctx.lineTo(x + radius[3], y + h);
+        ctx.arcTo(x, y + h, x, y + h - radius[3], radius[3]);
+        ctx.lineTo(x, y + radius[0]);
+        ctx.arcTo(x, y, x + radius[0], y, radius[0]);
+        ctx.closePath();
+
+        if (fill) {
+            if (Utils.isNumber(fillColor)) {
```

Comment:
Missing Utils namespace

Reply:
Updated the code based on your feedback and suggestion.